plugin/trino-hive/src/main/java/io/trino/plugin/hive/util/RetryDriver.java

```diff
@@ -50,15 +50,15 @@ private RetryDriver(
             Duration maxSleepTime,
             double scaleFactor,
             Duration maxRetryTime,
-            List<Class<? extends Exception>> allowedExceptions,
+            List<Class<? extends Exception>> retryStoppedExceptions,
             Optional<Runnable> retryRunnable)
     {
         this.maxAttempts = maxAttempts;
         this.minSleepTime = minSleepTime;
         this.maxSleepTime = maxSleepTime;
         this.scaleFactor = scaleFactor;
         this.maxRetryTime = maxRetryTime;
-        this.allowedExceptions = allowedExceptions;
+        this.retryStoppedExceptions = retryStoppedExceptions;
```

Comment:
I would argue that `stopOnExceptions`  brings more meaning compared to `retryStoppedExceptions`.
See also the method `io.trino.plugin.hive.util.RetryDriver#stopOn`